Datadog.Profiler.sln

```diff
@@ -58,9 +59,6 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Managed", "Managed", "{D22CAFC0-1633-4D87-BBE9-10F8D8401382}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "test", "test", "{13B8D121-5C5E-4998-8360-B4B214EDD23C}"
-	ProjectSection(SolutionItems) = preProject
-		test\Directory.Build.props = test\Directory.Build.props
-	EndProjectSection
```

Comment:
I don't know if this one should have gone? 🤔 But also, meh, who knows with .sln files 😅